src/gate.h

```diff
@@ -167,19 +167,31 @@ const utils::Complex nop_c[] = {
     0.0, 1.0
 };
 
-
+/*
+ * additional definitions for describing conditional gates
+ */
+typedef enum e_cond_type {
+    // 0 operands:
+    cond_always, cond_never,
+    // 1 operand:
+    cond_unary, cond_not,
```

Comment:
giving it some more thought, it remains tough: "cond" is a prefix, and the second part is a Boolean operation. In that sense the original "cond" is correct, but on it's own it doesn't 'feel' logical. "cond_unary" uses a different system, and "not" is also unary. 
I feel that the Python interface could better be without the "COND"-part, and then an empty string as operation does make sense.